<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>640</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>640</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page_my">
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>640</height>
     </size>
    </property>
    <widget class="QLabel" name="my_label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>441</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>--------------------------------------------------------------------------</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_my">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>我的</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_shouCang">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>收藏夹</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_video">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>视频</string>
     </property>
    </widget>
    <widget class="QLabel" name="my_label_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>我的音乐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_bang">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>榜单</string>
     </property>
    </widget>
    <widget class="QLabel" name="my_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>我创建的歌单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_music">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>580</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_like">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>   最喜欢的音乐</string>
     </property>
     <property name="autoRepeatDelay">
      <number>299</number>
     </property>
    </widget>
    <widget class="QPushButton" name="my_jingXuan">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>精选</string>
     </property>
    </widget>
    <widget class="QLabel" name="my_label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>用户：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_bengDi">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>本地音乐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_zuiJing">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>最近播放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_return">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="my_download">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>下载管理</string>
     </property>
    </widget>
    <widget class="QLabel" name="my_name">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="my_shouSuo">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>22</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_bengDiList">
    <widget class="QListWidget" name="bengDi_list">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>331</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bengDi_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>本地播放列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bengDi_addMusic">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加音乐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bengDi_my">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>我的</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bengDi_jingXue">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>精选</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bengDi_music">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>580</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bengDi_bangDan">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>榜单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bengDi_shiPing">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>视频</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_music">
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>640</height>
     </size>
    </property>
    <widget class="QPushButton" name="music_return">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="playBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>550</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="music_name">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>21</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>500</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>21</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>570</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>单曲循环</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>随机播放</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="music_up">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>562</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>上一曲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="music_down">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>562</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>下一曲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="music_list">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>570</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>列表</string>
     </property>
    </widget>
    <widget class="QLabel" name="music_text">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>341</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="music_photo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_bang">
    <widget class="QPushButton" name="bang_my">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>我的</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_40">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>精选</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bang_music">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>580</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bang_bang">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>榜单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bang_video">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>590</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>视频</string>
     </property>
    </widget>
    <widget class="QLabel" name="bang_label_1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>排行榜</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_set">
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>640</height>
     </size>
    </property>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>640</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>640</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_set_2">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>640</height>
       </size>
      </property>
      <widget class="QLabel" name="set_label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>441</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>--------------------------------------------------------------------------</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_shouSuo">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>22</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_my">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>590</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>我的</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_shouCang">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>250</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>收藏夹</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_video">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>590</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>视频</string>
       </property>
      </widget>
      <widget class="QLabel" name="set_label_1">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>我的音乐</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_bang">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>590</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>榜单</string>
       </property>
      </widget>
      <widget class="QLabel" name="set_label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>我创建的歌单</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_music">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>580</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>音乐</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_like">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>410</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string notr="true">   最喜欢的音乐</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_jingXuan">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>590</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>精选</string>
       </property>
      </widget>
      <widget class="QLabel" name="set_label_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>用户：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_bengDi">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>本地音乐</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_zuiJin">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>最近播放</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_return_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>——</string>
       </property>
      </widget>
      <widget class="QPushButton" name="set_download">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>下载管理</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>181</width>
         <height>561</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>179</width>
          <height>559</height>
         </rect>
        </property>
        <widget class="QPushButton" name="set_return">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>51</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
        <widget class="QPushButton" name="set_qianDao">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>每日签到</string>
         </property>
        </widget>
        <widget class="QPushButton" name="set_sleep">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>睡眠模式</string>
         </property>
        </widget>
       </widget>
      </widget>
      <zorder>set_music</zorder>
      <zorder>set_label_3</zorder>
      <zorder>set_shouSuo</zorder>
      <zorder>set_my</zorder>
      <zorder>set_shouCang</zorder>
      <zorder>set_video</zorder>
      <zorder>set_label_1</zorder>
      <zorder>set_bang</zorder>
      <zorder>set_label_4</zorder>
      <zorder>set_like</zorder>
      <zorder>set_jingXuan</zorder>
      <zorder>set_label_2</zorder>
      <zorder>set_bengDi</zorder>
      <zorder>set_zuiJin</zorder>
      <zorder>set_return_1</zorder>
      <zorder>set_download</zorder>
      <zorder>textBrowser_4</zorder>
      <zorder>scrollArea</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
